### DISCUSSION TITLE
No margins around tables in "readme" tab.

### DISCUSSION THREAD

**Discussion by iliubinskii** [npm/feedback#431] [Website] [answered]
As a result table border merges with horizontal rules related to titles.

Is there any way to improve styling of readme.md files?

**Comment by MylesBorins** [comment]
@ilyub can you please share an example of a table with poor rendering?

**Comment by iliubinskii** [comment]
Example:

https://www.npmjs.com/package/@typelib/eslint-plugin

readme.md:
```
#### Options

| Name | Description |
| :------ | :------ |
| `auto`  | Extracts identifiers from file. |
| `identifier`  | Identifier to be imported. |
| `originalIdentifier`  | Original identifier located in the imported file. |
| `source`  | Import source. |
| `starImport`  | Import identifier as namespace. |

### <a name="disallow-by-regexp"></a>disallow-by-regexp
```

I would expect some space between rules and table border. Instead they merge at the bottom and nearly merge at the top. See screenshot:

![Example](https://user-images.githubusercontent.com/70687432/122177636-78a9bd80-ce8e-11eb-8d36-44da7c616c64.jpg)

**Reply by MylesBorins** [reply]
I've filed a ticket with engineering. Thank you

**Comment by MylesBorins** [comment, marked as answer]
I believe this is fixed, please lmk if this isn't the case
